src/Microsoft.VisualStudio.Threading.Analyzers.Tests/VSTHRD109AvoidAssertInAsyncMethodsAnalyzerTests.cs

```diff
@@ -252,11 +247,10 @@ class Test {
 }
 ";
 
-            var expected = Verify.Diagnostic().WithSpan(9, 26, 9, 46);
-            await Verify.VerifyCodeFixAsync(test, expected, fix);
+            await Verify.VerifyCodeFixAsync(test, fix);
         }
 
-        [Fact(Skip = "Fails because the semantic model claims the anonymous func returns Task instead of Task<int>. We should probably skip that check and always preserve return statements.")]
```

Comment:
📝 This was caused by the use of a semantic model from an intermediate state - the method was not yet marked as `async` but an `await` expression had already been added.